packages/rollup/test/integration/BUILD.bazel

```diff
@@ -56,12 +58,46 @@ nodejs_binary(
     entry_point = ":sha256.js",
 )
 
+# Verify the hash of bundle .map files
```

Comment:
I added a commit to test the `.map` file don't change similar to the `.js` tests. Not sure how valuable that is? But right now I'm getting errors with sourcemaps being inlined instead of in their own files so I wanted to verify the `.map`s were being generated correctly...